seed/php-sdk/imdb/clientName/src/Core/Client/RetryMiddleware.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Seed\Core\Client;
+
+use GuzzleHttp\Exception\ConnectException;
+use GuzzleHttp\Promise as P;
+use GuzzleHttp\Promise\PromiseInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Throwable;
+
+class RetryMiddleware
+{
+    private const RETRY_STATUS_CODES = [408, 429];
+
+    /**
+     * @var callable(RequestInterface, array): PromiseInterface
+     * @phpstan-ignore missingType.iterableValue
+     */
+    private $nextHandler;
```

Comment:
Thoughts on shortening this to just `$next`? I've seen this naming convention used elsewhere, but not married to it if you feel strongly otherwise.

Reply:
A large amount of the code is copied from the built-in RetryMiddleware from the Guzzle library itself, so I think this is probably a good name